vault/templates/job-eso.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if .Values.esoInit.enabled -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Chart.Name }}-eso-init
```

Comment:
what's the point of having multiple templates for jobs? I don't see much difference between them, so I think we may combine them into a single job template and instead of using `{{- if .Values.esoInit.enabled -}}` we can just iterate the jobs object